install_config/configuring_authentication.adoc

```diff
@@ -1665,8 +1665,8 @@ with a non-empty value is used. The
 link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[standard claims] are:
 
 [horizontal]
-`sub`:: The user identity.
-`preferred_username`:: The preferred user name when provisioning a user.
+`sub`:: The ID of the user within the configured identity provider. For example `user1`. This can be created before the authorization process. When added to the identity object, this is prefixed by the identity provider object. For example `myidp:user1`. If the identity object does not exist, so, if the user has not been created, then the prefix depends on the `preferred_username`.
```

Comment:
This description doesn't make sense here… this is describing standard claims that are part of the open id spec, not users you create.